tests/integration/ceph_base_object_test.go

```diff
@@ -136,7 +144,8 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	s3endpoint, _ := helper.ObjectClient.GetEndPointUrl(namespace, storeName)
 	s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
 	s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
-	s3client := utils.CreateNewS3Helper(s3endpoint, s3AccessKey, s3SecretKey)
+	s3client, err := rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint)
```

Comment:
Shall we remove this whole section of the test that creates a bucket and performs the PUT/GET? That will already be performed by the health check now.

Reply:
Yes, since the check for Healthy status :)

Reply:
Actually, this is testing OBC so we should keep it.